config.json

```diff
@@ -25,9 +25,7 @@
     "redis": {
       "host": "127.0.0.1",
       "port": 6380,
-      "db": 0,
-      "user": "",
-      "password": "vgeuwrKZLYwNyZXNc7vTFbjJ4vQnMGrs"
```

Comment:
Why are we removing this from here? Can we not just delete the entries from the object when null during connect?